node/resolvers/Queries/Users.ts

```diff
@@ -407,11 +407,15 @@ const Users = {
         return null
       })
 
-      if (availableSalesChannels.length) {
-        access =
-          selectedChannels?.filter((item: any) =>
-            availableSalesChannels?.includes(item)
-          ).length > 0
+      if (selectedChannels) {
```

Comment:
Could this be simplified with the line bellow? 

```
if(something && somethingElse) {
  Do something
}

Reply:
Not really here, we are in the structure like 
if (A) {
  if (B) { C}
} else {
 D
}